types/coin.go

```diff
@@ -270,6 +282,10 @@ func (coins Coins) Add(coinsB ...Coin) Coins {
 // denomination and addition only occurs when the denominations match, otherwise
 // the coin is simply added to the sum assuming it's not zero.
 func (coins Coins) safeAdd(coinsB Coins) Coins {
```

Comment:
I'd update godoc to indicate a panic occurs if the coins are not sorted

Reply:
ok